servers/store/src/main/proto/table.proto

```diff
@@ -41,9 +41,12 @@ message IcebergRefState {
   int32 sort_order_id = 4;
 }
 
-message SqlView {
+message IcebergView {
```

Comment:
IIUC Iceberg views correctly, `versionId` is similar to `snapshotId` in Iceberg tables and
Iceberg views have multiple versions of a view, analogous to multiple snapshot in Iceberg tables.

I also suspect that there should always be just one current "view-metadata-json" (analogous to one current "table-metadata-jsoin").

So I think you have to add glolbal state to Iceberg views like global state for Iceberg tables - i.e. store the current "view-metadata-json" via `IcebergMetadataPointer`.

/cc @rymurr